R/projects.R

```diff
@@ -8,18 +8,19 @@
 #' @export
 space_project_list <- function(space, filters = NULL) {
   space_id <- space_id(space)
+  space_project_list_url <- paste0("/spaces/", space_id, "/content/projects")
+
   query_list <- filters %>%
     purrr::map(~ list("filter" = .x)) %>%
-    append(list("filter" = paste0("space_id:", space_id))) %>%
     purrr::flatten()
 
-  response <- rscloud_rest("projects",
-    query = query_list
+  response <- rscloud_rest(space_project_list_url,
+                           query = query_list
   )
 
   verify_response_length(response, "projects", filters)
 
-  projects <- collect_paginated(response, path = "projects", query = query_list)
+  projects <- collect_paginated(response, path = space_project_list_url, query = query_list)
```

Comment:
I've tried this with a few of my spaces and I'm getting the same error regardless of whether the space has LOTS of projects or not so many. The issue seems to be that `projects` here returns a list of `NULL`s and so the subsequent pipeline fails.